src/include/txn.h

```diff
@@ -242,6 +242,22 @@ struct __wt_txn_op {
     uint32_t flags;
 };
 
+/*
+ * WT_TXN_SNAPSHOT_DATA --
+ *	A structure to store the transactions snapshot details.
+ */
+struct __wt_txn_snapshot_data {
```

Comment:
my suggestion is to name the struct as __wt_txn_snapshot, just remove _data at the end.

Reply:
Fixed in 3523185a21d3fff8e71151f767b27376ee91d929